### PR TITLE
Fix scrubber with insecure ssl

### PR DESCRIPTION
**Pull request by vuntz**
Backport of https://github.com/crowbar/crowbar-openstack/pull/306

### CODE REVIEW

File: chef/cookbooks/glance/templates/default/glance-cache.conf.erb [hunk 1]
```diff
@@ -31,6 +31,9 @@ registry_host = <%= @registry_bind_host %>
 # Port the registry server is listening on
 registry_port = <%= @registry_bind_port %>
 
+<% # Missing in upstream template -%>
```
Comment (aplanas) [inline chef/cookbooks/glance/templates/default/glance-cache.conf.erb:34]: ouch, a fail in the package?

**Comment by aplanas** [comment]
+1

File: chef/cookbooks/glance/templates/default/glance-cache.conf.erb (hunk 1 above)
Comment (vuntz) [inline chef/cookbooks/glance/templates/default/glance-cache.conf.erb:34]: No upstream bug, fixed in Mitaka (iirc): glance config files are not automatically generated upstream.

**Comment by jsuchome** [comment]
+1

**Pull request merged**